airflow/operators/hive_to_druid.py

```diff
@@ -78,7 +78,7 @@ def __init__(
             target_partition_size=-1,
             query_granularity="NONE",
             segment_granularity="DAY",
-            hive_tblproperties=None,
+            hive_tblproperties=dict(),
```

Comment:
https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

Reply:
Better idea is write:
```
        self.hive_tblproperties = hive_tblproperties or None
```
instead
````
        self.hive_tblproperties = hive_tblproperties
````

Reply:
I modified it according to mik-laj's advice. 03252c86e446b3e81c0690c820b069177dc42a7d